FrontEnd/src/components/basics/ImageDropdown/ImageDropdown.tsx

```diff
@@ -50,7 +46,7 @@ const ImageDropdown: FunctionComponent<itemProps> = ({
               className={cn(
                 labelClassName,
                 'text-center',
-                router.pathname.toLowerCase().slice(1) === label.toLowerCase()
+                  selected
```

Comment:
A better way to do this is 
selected && styles.isSelected
or
{
  [styles.isSelected]: selected
}